sample/adder.py

```diff
@@ -0,0 +1,105 @@
+"""
+SAMPLE PROGRAM : 5-bit adder
+
+REFERENCE : Figure 1 in C. Gidney, arXiv:1709.06648
+"""
+
+
+from quantestpy import (QuantestPyCircuit,
+                        assert_circuit_equivalent_to_output_qubit_state)
+
+"""Define a function testing the circuit"""
+
+
+def test_5_bit_adder(circuit):
+    """a + b = c"""
+
+    for decimal_a in range(2**5):
+        for decimal_b in range(2**5):
+            bitstring_a = ("0" * 5 + bin(decimal_a)[2:])[-5:]
+            bitstring_b = ("0" * 5 + bin(decimal_b)[2:])[-5:]
+            decimal_c = decimal_a + decimal_b
+            if decimal_c > 31:
+                continue
```

Comment:
`continue` can be replace to the line before line 19.

Reply:
Done in https://github.com/QuantestPy/quantestpy/pull/198/commits/f7e800f85e0e867a96482122358e2e5132b64a92